scene/3d/node_3d.cpp

```diff
@@ -916,6 +995,9 @@ void Node3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("to_local", "global_point"), &Node3D::to_local);
 	ClassDB::bind_method(D_METHOD("to_global", "local_point"), &Node3D::to_global);
 
+	ClassDB::bind_method(D_METHOD("property_can_revert"), &Node3D::property_can_revert);
+	ClassDB::bind_method(D_METHOD("property_get_revert"), &Node3D::property_get_revert);
```

Comment:
You need to define the argument name:
```suggestion
	ClassDB::bind_method(D_METHOD("property_can_revert", "name"), &Node3D::property_can_revert);
	ClassDB::bind_method(D_METHOD("property_get_revert", "name"), &Node3D::property_get_revert);
```